arkouda/strings.py

```diff
@@ -840,5 +860,7 @@ def unregister(self) -> None:
 
     @staticmethod
     def attach(user_defined_name : str) -> Strings:
-        return Strings(pdarray.attach(user_defined_name+'_offsets'),
+        s = Strings(pdarray.attach(user_defined_name+'_offsets'),
                        pdarray.attach(user_defined_name+'_bytes'))
+        s.name = user_defined_name
+        return s
```

Comment:
This change was requested by @glitch. The rationale is if the client is disconnected from the server, then `Strings.name` would've been wiped out and should be re-declared when we attach

Reply:
👍 good